package.json

```diff
@@ -1,9 +1,10 @@
 {
-  "name": "lightning-ui-components",
-  "version": "0.0.0",
+  "name": "@lightningjs/ui-components",
+  "version": "0.0.1",
```

Comment:
should we bump this to 1.0.0 before we publish?

Reply:
Once both PRs are in I will.